### PR TITLE
CommonStyling to chapters

### PR DESCRIPTION
**Pull request by jkc9886**
Common styling to package, function and objects notations. Observed grammar errors corrected.

### CODE REVIEW

**Comment by antagomir** [comment]
Can you resolve the conflicts? Otherwise seems good!

**Review by antagomir — APPROVED** [review]

**Comment by jkc9886** [comment]
Sure, I am on it!

**Comment by antagomir** [comment]
ready?

**Comment by jkc9886** [comment]
Yes, this PR is ready to be merged.

File: inst/pages/extra_material.qmd [hunk 1]
```diff
@@ -678,9 +679,9 @@ Here, we use following packages:
 -   [biclust](https://cran.r-project.org/web/packages/biclust/index.html)
 -   [cobiclust](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13582)
 
-_cobiclust_ is especially developed for microbiome data whereas _biclust_ is
-more general method. In this section, we show two different cases and example
-solutions to apply biclustering to them.
+`_cobiclust_` is especially developed for microbiome data whereas `_biclust_` is
```
Comment (TuomasBorman) [inline inst/pages/extra_material.qmd:682]: These packages should be

\`cobiclust\` and not \`\_cobiclust\_\`

There were couple of lines were this same occurred. Fix them all

File: inst/pages/introductory_workflow.qmd [hunk 2]
```diff
@@ -146,7 +146,7 @@ manipulation using [**the dplyr package**](https://dplyr.tidyverse.org/).
 ### Subsetting
 
 In some cases, you may only need to use part of your original
-*TreeSummarizedExperiment*.
+`*TreeSummarizedExperiment*`.
```
Comment (TuomasBorman) [inline inst/pages/introductory_workflow.qmd:149]: This should be \`TreeSummarizedExperiment\` without additional styling. Check line 107 also

**Comment by TuomasBorman** [comment]
Looks good! Check comments

File: inst/pages/extra_material.qmd (hunk 1 above)
Comment (jkc9886) [reply inst/pages/extra_material.qmd:682]: Sure, the underscores were there initially so I thought not to remove them. Maybe they were also formatting errors, I shall do the needful.

File: inst/pages/introductory_workflow.qmd (hunk 2 above)
Comment (jkc9886) [reply inst/pages/introductory_workflow.qmd:149]: Sure.

File: inst/pages/extra_material.qmd (hunk 1 above)
Comment (TuomasBorman) [reply inst/pages/extra_material.qmd:682]: Yep, we have used different styles previously, but now we should use harmonized style everywhere

Comment (jkc9886) [reply inst/pages/extra_material.qmd:682]: Hello, I am no longer able to make changes to these files, should I make the corrections ina new PR?

Comment (antagomir) [reply inst/pages/extra_material.qmd:682]: Yes you can do that, probably easier now as this was merged.

**Pull request merged**